CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2201**
   - Update `dbIndexer` lambda to process requests in serial
   - Fixes ingestPdrWithNodeNameSpec parsePdr provider error
+- **CUMULUS-2251**
+  - Move Egress Api Gateway Log Group Filter from `distribution/main.tf` to `/cumulus-tf/main.tf` and remove `tea_api_egress_log_group` variable from `distribution/variables.tf` and `cumulus/variables.tf` and add `log_api_gateway_to_cloudwatch` variable to `cumulus-tf/variables.tf`
```

Comment:
It would be good to split these notes about what was changed, removed, and added into separate notes under the different headers for Changed, Added, etc.

Reply:
We also need to add a migration note here - since people will be expected to add the new resource definition to their cumulus-tf/main.tf file. Also, we need to update the [template-deploy repo](https://github.com/nasa/cumulus-template-deploy/pulls) that we maintain as a starter/example for users with these changes.